dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/fat/src/com/ibm/ws/jaxrs20/cdi12/fat/test/ContextandCDI12Test.java

```diff
@@ -0,0 +1,205 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxrs20.cdi12.fat.test;
+
+import static org.junit.Assert.assertTrue;
```

Comment:
```suggestion
import static org.junit.Assert.assertEquals;
```

This should make a better failure message if the verifySuccess method fails.